src/com/company/FootballMatch.java

```diff
@@ -1,11 +1,11 @@
 package com.company;
-import java.util.Scanner; // импорт сканера
+import java.util.Scanner; 
 
 
 public class FootballMatch {
 
     static Scanner scan = new Scanner(System.in);
-    public int MatchOutcome; // переменная исхода матча
+    public int MatchOutcome; 
 
     public int getInt(){//валидатор ввода данных
```

Comment:
if you need write the comment you must use English language. This comment is unnecessary there.

Reply:
try to use English to write message in System.out.println()

Reply:
move getInt() in the main class or in the another class like Service. This code repeat in all classes.